[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[dev-packages]
mypy = "==0.620"
pytest = "==3.7.3"

# DO NOT UPGRADE, latest version breaks paths on Windows:
# https://github.com/pytest-dev/pytest-django/issues/641
pytest-django = "==3.3.3"

"flake8" = "==3.5.0"
pytest-cov = "==2.5.1"
python-dotenv = "==0.9.1"
factory-boy = "==2.11.1"
requests-mock = "==1.5.2"
django-webtest = "==1.9.3"
freezegun = "==0.3.11"

[packages]
django = "==2.1.2"
whitenoise = "==4.0"
dj-database-url = "==0.5.0"
django-csp = "==3.4"
gunicorn = "==19.9.0"
"psycopg2-binary" = "==2.7.5"
graphene-django = "==2.1.0"
pymongo = "==3.7.1"
pydantic = "==0.13"
requests = "==2.20.0"
rollbar = "==0.14.5"
twilio = "==6.18.0"

[requires]
python_version = "3.7"
